src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/queue/QueueService.kt

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.hmpps.personrecord.service.queue
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.stereotype.Component
+import software.amazon.awssdk.services.sns.model.PublishRequest
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
+import uk.gov.justice.digital.hmpps.personrecord.client.model.court.MessageType
+import uk.gov.justice.digital.hmpps.personrecord.client.model.sqs.NOTIFICATION
+import uk.gov.justice.digital.hmpps.personrecord.client.model.sqs.messages.Recluster
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+import uk.gov.justice.hmpps.sqs.MissingQueueException
+import uk.gov.justice.hmpps.sqs.MissingTopicException
+import java.util.UUID
+import software.amazon.awssdk.services.sns.model.MessageAttributeValue as SNSMessageAttribute
+import software.amazon.awssdk.services.sqs.model.MessageAttributeValue as SQSMessageAttribute
+
+@Component
+class QueueService(
+  private val objectMapper: ObjectMapper,
+  private val hmppsQueueService: HmppsQueueService,
```

Comment:
Can split into `QueueService` and `TopicService` which aligns more of what they are doing, but since the topic logic is only temporary. Is it worth the extra logic for now?